sdk/spring/spring-cloud-azure-autoconfigure/pom.xml

```diff
@@ -317,6 +317,12 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-test</artifactId>
+      <version>2.5.4</version> <!-- {x-version-update;org.springframework.boot:spring-boot-test;external_dependency} -->
+      <scope>test</scope>
+    </dependency>
```

Comment:
already included in the spring-boot-starter-test?

Reply:
Deleted.